linux/net/rina/rmt.h

```diff
@@ -59,6 +59,7 @@ struct rmt;
 enum flow_state {
 	N1_PORT_STATE_ENABLED,
 	N1_PORT_STATE_DISABLED,
+	N1_PORT_STATE_DO_NOT_DISABLE,
```

Comment:
What do you mean? 

All the states are an implementation issue. It is the only way to solve the race condition, do you have a better alternative?